spyne/server/twisted/http.py

```diff
@@ -183,7 +183,7 @@ def _reconstruct_url(request):
         url_scheme = 'http'
 
     uri = _decode_path(request.uri)
-    return ''.join([url_scheme, "://", server_name, uri])
+    return ''.join([url_scheme, "://", server_name, uri.decode('utf-8')])
```

Comment:
How do we know that uri is encoded with utf-8 here?

Reply:
I thought it was assumed as we create a "context" with utf-8 content type at line 676, but it concerns more the content than the headers.
The standard doesn't seem to allow other chars than ascii in URI during transmission, maybe it's better to decode as ascii rather than utf-8